src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Applications/_TabularData.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@using Newtonsoft.Json
+@model dynamic
+
+@{
+    var tableData = JsonConvert.DeserializeObject<dynamic>(Model);
```

Comment:
Can we do anything to move away from dynamic types? I'm presuming not :(

Reply:
I'd rather not deviate from the view used in QnA/Apply